locale/en/blog/vulnerability/v8-memory-corruption-stack-overflow.md

```diff
@@ -41,15 +41,15 @@ index b90191d..2769ca7 100644
   public:
    explicit StackLimitCheck(Isolate* isolate) : isolate_(isolate) { }
 
--  bool HasOverflowed() const {
+*  bool HasOverflowed() const {
 +  inline bool HasOverflowed() const {
      StackGuard* stack_guard = isolate_->stack_guard();
--    // Stack has overflowed in C++ code only if stack pointer exceeds the C++
--    // stack guard and the limits are not set to interrupt values.
--    // TODO(214): Stack overflows are ignored if a interrupt is pending. This
--    // code should probably always use the initial C++ limit.
--    return (reinterpret_cast<uintptr_t>(this) < stack_guard->climit()) &&
--           stack_guard->IsStackOverflow();
+*    // Stack has overflowed in C++ code only if stack pointer exceeds the C++
```

Comment:
This is a `diff`. Not sure if there are more cases like this. Which reminds me maybe specify `diff` in the codeblock lang